yarn.lock

```diff
@@ -447,9 +447,9 @@
   integrity sha512-yALhelO3i0hqZwhjtcr6dYyaLoCHbAMshwtj6cGxTvHZAKXHsYGdff6E8EPw3xLKY0ELUTQ69Q1rQiJENnccMA==
 
 "@types/jest@^24.0.13", "@types/jest@^24.0.6":
-  version "24.0.14"
```

Comment:
Don't update the dependencies. Restore the original content of yarn.lock.

Reply:
Seems like a merge gone wrong; needs a rebase (@Bugvi-Benjamin-M).